Makefile

```diff
@@ -66,7 +66,8 @@ uninstall:
 # Run static analysis.
 .PHONY: verify
 verify:
-	hack/verify-all.sh
+	go mod vendor
```

Comment:
It's usually bad form to have the Makefile create potentially version controller files.
Can we wrap this using docker so the files downloaded are temporary?

Reply:
Or we should gitignore and make clean vendor?

Reply:
Using `-mod=readonly` removes the need to populate vendor.